mon-pix/app/routes/campaigns/start-or-resume.js

```diff
@@ -20,25 +31,22 @@ export default BaseRoute.extend({
 
   afterModel() {
     const store = this.get('store');
-
-    if (this._userIsUnauthenticated()) {
-      return this.transitionTo('campaigns.campaign-landing-page', this.get('campaignCode'));
-    }
-
     return store.query('assessment', { filter: { type: 'SMART_PLACEMENT', codeCampaign: this.get('campaignCode') } })
       .then((smartPlacementAssessments) => {
 
-        if (this._thereIsNoAssessment(smartPlacementAssessments)) {
+        if (this._thereIsNoAssessment(smartPlacementAssessments) && this.get('userHasSeenLanding')) {
```

Comment:
Je veux bien tenter ensemble de refacto ces if (deux conditions, ça me fait un peu peur).
Pourquoi les imbriquer, mais la c'est la forêt de if qui me ferait peur. Du coup, chaud pour réfléchir ensemble.